GoogleTestNuGet/Build.ps1

```diff
@@ -192,7 +192,7 @@ function Build-Binaries {
     Push-Location $Dir
     try {
         $CMakeArgs = @()
-        $CMakeArgs += "-G", "Visual Studio 15 2017"
+        $CMakeArgs += "-G", "`"Visual Studio 15 2017`""
```

Comment:
This came from this error message:
`Invoke-Executable failed for "cmake -G Visual Studio 15 2017 -T v141 -A Win32 -D BUILD_SHARED_LIBS=OFF -D
gtest_force_shared_crt=ON -D CMAKE_CXX_FLAGS=/D_SILENCE_TR1_NAMESPACE_DEPRECATION_WARNING
D:\Code\TestAdapterForGoogleTest\GoogleTestNuGet\..\ThirdParty\googletest\googletest" with working directory
"D:\Code\TestAdapterForGoogleTest\GoogleTestNuGet\Intermediate\build.v140.windesktop.msvcstl.static.rt-dyn.Win32".`
And when I copy-pasted the line, it failed due to the lack of quoting. But now that I think about it, it's probably just an error with the error message formatting because this is supposed to build in CI. I'll remove them.